language/move-prover/src/bytecode_translator.rs

```diff
@@ -810,7 +969,11 @@ impl<'env> ModuleTranslator<'env> {
         emitln!(self.writer, "\n// bytecode translation starts here");
 
         // Generate bytecode
-        let code = func_target.get_bytecode();
+        let code = if let Some(bytecode) = rewritten_bytecode {
+            &bytecode
```

Comment:
See above. We should not need to carry rewritten bytecode around, it should be in the FunctionTarget as result of the transformation pipeline.